lxd/db/cluster/identities.go

```diff
@@ -574,6 +574,26 @@ WHERE auth_groups.name IN %s
 	}
 
 	if int(rowsAffected) != len(groupNames) {
+		// No longer on happy "fast" path.
+		// Check each group name exists to return a nice error
+		missingGroups := make([]any, 0, len(groupNames))
```

Comment:
Why slice of any?

Reply:
This was to pass into the `NewStatusError` function as it requires a slice of any. I've changed it now to use strings.Join and string concatenation rather than a format string.